src/main/java/org/kohsuke/github/GHReason.java

```diff
@@ -0,0 +1,34 @@
+package org.kohsuke.github;
+
+/**
+ * The possible values for reason in verification object from github.
+ * 
+ * @see <a href="https://developer.github.com/v3/repos/commits/#signature-verification-object">List of possible reason
+ *      values</a>
+ * @author Sourabh Sarvotham Parkala
+ */
+public enum GHReason {
+    EXPIRED_KEY("expired_key"),
```

Comment:
We need at test for each of these values. Or at least a bunch of them.  Even if you have to manually create them.